templates/server/src/main/resources/archetype-resources/core/src/main/java/__packageInPathFormat__/general/common/impl/config/ApplicationObjectMapperFactory.java

```diff
@@ -27,11 +25,8 @@ public ApplicationObjectMapperFactory() {
 
     super();
     // see https://github.com/devonfw-wiki/devon4j/wiki/guide-json#json-and-inheritance
-    SimpleModule module = getExtensionModule();
+    SimpleModule module = super.initMapping();
```

Comment:
when `initMapping()` is called from constructor, we can keep the original `getExtensionModule()` line.

Reply:
Done